platform/qemu.go

```diff
@@ -76,10 +77,45 @@ func MakeCLDiskTemplate(inputPath string) (output *os.File, result error) {
 		return nil, fmt.Errorf("copying file: %v", err)
 	}
 
+	if extraSize != "" {
+		resizeRawDisk(outputPath, extraSize)
+	}
+
+	if logSetting == EnableConsoleLog {
+		enableConsoleLog(outputPath)
+	}
+
+	// return fd to output file
+	output, err := os.Open(outputPath)
+	if err != nil {
+		return nil, fmt.Errorf("opening %v: %v", outputPath, err)
+	}
+	return output, nil
+}
+
+func resizeRawDisk(disk, extraSize string) error {
+	// Resize the base disk, booting the image should claim the
+	// free space.
+	resize := exec.Command("qemu-img", "resize", "-f", "raw", disk,
```

Comment:
```suggestion
func resizeDisk(disk, extraSize string) error {
	// Resize the base disk, booting the image should claim the
	// free space.
	resize := exec.Command("qemu-img", "resize", disk,
```
This works for both raw and qcow2